packages/manager/modules/manager-components/src/index.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
No other solution than to disable eslint ?

Reply:
You could do: 
```
import { default as managerResourceSelector } from './resource-selector`
export default managerResourceSelector;
```

Reply:
It seems to be the same. I will have to add `/* eslint-disable import/no-named-default */` if I go with this suggestion :)

Reply:
I will have to add `/* eslint-disable import/no-named-default */` if I use the above suggestion... I think it is more or less the same....

Reply:
@ganeshkumar1989, to avoid to add an ESLINT exception, i think we can just import it like this:
```suggestion
export managerResourceSelector from './resource-selector';
```

For info, when you import something that is exported with `default` keyword, we can rename it like we would.
I hope it will helps.

Reply:
This does not seem to work either... I get an error "Declaration or statement expected"...

Reply:
```js
import { default as managerResourceSelector } from './resource-selector` 
```
will cause eslint `import/no-named-default` error, because you don't have to destructure the `default` when importing.


something similar to @JeremyDec 's proposal should works : 
```js
import resourceSelector from './resource-selector` 
export default resourceSelector;
```

Reply:
This is done now. Thanks a lot for the suggestions/help